test/account_test.rb

```diff
@@ -150,13 +150,14 @@ def app
       it "redirects back to reset page when there's a #{error_code} error" do
         stub_heroku_api do
           post("/auth/finish_reset_password/c45685917ef644198a0fececa10d479a") {
-            [ error_code, MultiJson.encode({ message: "a #{error_code} error" }) ]
+            [ error_code, [MultiJson.encode({ message: "a #{error_code} error" })] ]
```

Comment:
Line is too long. [84/80]<br>Redundant curly braces around a hash parameter.<br>Space inside square brackets detected.